nixos/modules/services/x11/display-managers/default.nix

```diff
@@ -306,7 +307,7 @@ in
           description = "Additional environment variables needed by the display manager.";
         };
 
-        logsXsession = mkOption {
+        logToFile = mkOption {
```

Comment:
I think you'll have to add something like
```
mkRenamedOptionModule [ "services" "xserver" "displayManager" "job" "logsXsession" ] [ "services" "xserver" "displayManager" "job" "logsToFile" ];
```
to [rename.nix](https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/rename.nix). Otherwise backwards compatibility is broken.

Same with `logToJournal` below.

Reply:
That isn't a simple rename of `logsXsession` because of the logic bug.
That is something you need to take care of concering backwards compatibilty.

If `logsXsession` is `true`, `logToFile` needs to be `false` and vice versa to preserve the current behaviour.

Reply:
@rycee Thank you. It's done.
@Gerschtli The current behavior is wrong. It may be mentioned in the release changes but it makes no sense to keep it that way for compatibility.